### DISCUSSION TITLE
[Feature] Enable hiding from info side pane

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#6002] [Feature Request]
### The feature

Hi, I'll post my dream list of features today.

Here is the next one: Being able to hide/unhide a face directly from the side pane of a given asset.

Current User Experience:
- User opens a picture
- Realizes there are many useless faces that should be hidden
- Right-click -> Open in New Tab for all these faces
- Go over all the tabs to do "Menu Click" -> "Hide"
- Close all open tabs
- (Optional) Refresh initial page.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by LaurentAtJoes** [comment]
that would definitely come in very handy.
I have series of "group photos" (weddings, corporate events etc.), where I only want to tag a few people, and ignore (hide) all the rest.

**Comment by bo0tzz** [comment]
#5135
